docs/taskruns.md

```diff
@@ -58,6 +58,9 @@ following fields:
   - [`podTemplate`](#pod-template) - Specifies a subset of
     [`PodSpec`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/#pod-v1-core)
 	configuration that will be used as the basis for the `Task` pod.
+  - [`expirationSecondsTTL`] - Specifies expirationSecondsTTL at least zero which means that completed TaskRun will be deleted automatically. 
+    If `expirationSecondsTTL` is not set, TaskRun will not be deleted. And if a TaskRun is produced by PipelineRun, 
+    the value of expirationSecondsTTL of TaskRun will be ignored, then check PipelineRun's expirationSecondsTTL.
```

Comment:
I'm not sure I got this. 
If the `expirationSecondsTTL` can only be set on TaskRun and PipelineRun, when the TaskRun is created by a PipelineRun it cannot have it's own `expirationSecondsTTL` to be ignored.
The way I understand it is that any TaskRun created by a PipelineRun will inherit the `expirationSecondsTTL` value of the PipelineRun.

Reply:
Yes, "any TaskRun created by a PipelineRun will inherit the expirationSecondsTTL value of the PipelineRun." As we known, in this case, the field of TaskRun is from PipelineRun, so while PipelineRun is deleted after expired, taskRun will be deleted at the same time.